### PR TITLE
Add minikube role and install k8s apps inside minikube on sandbox

### PR DESCRIPTION
**Pull request by nadeemshahzad**
Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR if it is code shared across multiple services and you don't own all of the services.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).
  - [ ] Think about how this change will affect Open edX operators.  Have you updated the wiki page for the next Open edX release?

### CODE REVIEW

**Review by adzuci — APPROVED** [review]

**Review by edx-abolger — APPROVED** [review]
Hi @nadeemshahzad I appreciate this moving forward, but I also am not a technical expert in with Ansible, so I cannot meaningfully give you feedback on your PR at this time. I will happily give this a try as a role once it is merged.

File: util/jenkins/ansible-provision.sh [hunk 1]
```diff
@@ -521,5 +521,21 @@ if [[ $enable_newrelic == "true" ]]; then
...
 fi
 
+run_ansible run_role.yml -i "${deploy_host}," -e role=minikube $extra_var_arg  --user ubuntu
+
+if [[ $argocd == "true" ]]; then
+    # setup minikube on sandbox
+    run_ansible run_role.yml -i "${deploy_host}," -e role=minikube $extra_var_arg  --user ubuntu
+
+    kustomize build $WORKSPACE/edx-internal/argocd/applications/argocd/development -o argocd.yaml
+    kustomize build $WORKSPACE/edx-internal/argocd/clusters/development -o cluster.yaml
```
Comment (adzuci) [inline util/jenkins/ansible-provision.sh]:
```suggestion
    kustomize build $WORKSPACE/configuration-internal/argocd/applications/argocd/development -o argocd.yaml
    kustomize build $WORKSPACE/configuration-internal/argocd/clusters/development -o cluster.yaml
```

**Review by adzuci — CHANGES_REQUESTED** [review]
I think we can point at an existing checkout of edx-internal.

File: util/jenkins/ansible-provision.sh (hunk 1 above)
Comment (adzuci) [reply util/jenkins/ansible-provision.sh]: @coryleeio agrees.

File: util/jenkins/ansible-provision.sh [hunk 2]
```diff
@@ -573,5 +573,19 @@ if [[ $enable_newrelic == "true" ]]; then
...
 fi
 
+
+if [[ $license_manager == "true" ]]; then
+    k8s_apps="license_manager"
+fi
+if [[ $edx_notes_api == "true" ]]; then
+    k8s_apps+=" edx_notes_api"
+fi
+if [[ -z $k8s_apps ]]; then
```
Comment (adzuci) [inline util/jenkins/ansible-provision.sh]: Does k8s_apps get passed in somewhere currently?

Comment (adzuci) [reply util/jenkins/ansible-provision.sh]: Ah, I see it used here: https://github.com/edx/configuration/pull/5809/files#diff-8d11aca30e575de97b7a4436a10ea0a5031ff0b8f52c7e0eebdb746d760467d9R10

**Review by adzuci — APPROVED** [review]
:shipit:

**Pull request merged**